src/main/cli/cli.c

```diff
@@ -2706,6 +2706,44 @@ static void cliDisplayName(char *cmdline)
     }
     printDisplayName(DUMP_MASTER, pilotConfig());
 }
+
+static void printRateProfileName(uint8_t dumpMask, const profileName_t *rateProfileNameConfig)
+{
+    const bool equalsDefault = strlen(rateProfileNameConfig->name) == 0;
+    cliDumpPrintLinef(dumpMask, equalsDefault, "rateprofile_name %s", equalsDefault ? emptyName : rateProfileNameConfig->name);
+}
+
+static void cliRateProfileName(char *cmdline)
```

Comment:
Can you please wrap this into a `USE_PROFILE_NAMES` or suchlike define? This seems to be a nice to have feature, and should probably be left out of F3.

Reply:
Done.